pkgs/top-level/all-packages.nix

```diff
@@ -11947,11 +11947,22 @@ in
   gns3-gui = gns3Packages.guiStable;
   gns3-server = gns3Packages.serverStable;
 
-  gobject-introspection = callPackage ../development/libraries/gobject-introspection {
+  gobject-introspection-full-classic = callPackage ../development/libraries/gobject-introspection {
     nixStoreDir = config.nix.storeDir or builtins.storeDir;
     inherit (darwin) cctools;
   };
 
+  gobject-introspection-py-tools = gobject-introspection-full-classic.override {
```

Comment:
Can you explain the when to use what?

Reply:
Well I think I'll need to make a wrapper package so this isn't actually used directly. But the idea is (whether directly or via some wrapper) than the py tools are need when generating new gir files and type libs when building some C library, and aren't needed by the programs in python etc that just consume typelibs.

Reply:
I don’t think we should make that distinction if it’s not made by gobject introspection. IIUC things that consume gir also produce gir without much distinction.

Reply:
That's fine, but we will need the wrapper anyways to combine the build platform py tools with host platform everything else.